concepts/nlp_lexic/gold_entities.md

```diff
@@ -144,15 +144,15 @@ Keep an eye on it, as we're always improving the detection for current entities,
 {
   "local": "paul",
   "tag": null,
-  "domain": "recast.ai",
-  "raw": "paul@recast.ai",
+  "domain": "cai.tools.sap",
+  "raw": "paul@sap.com",
   "confidence": 0.99
 }
 ```
 
 | Entity | Examples |
 | ------ | -------- |
-| `email` | hello@recast.ai, hello+devs@recast.ai, hello.you+devs@recast.co.uk |
+| `email` | pierre-edouard.lieb@sap.com |
```

Comment:
It should be better to put a fake email (like myemail@sap.com for example) instead of P-E's email address :)